tests/file_tests/main.cpp

```diff
@@ -1,58 +1,37 @@
 #include <cstdlib>
 #include <string>
-#include <vector>
 
 #include "slang.h"
 
 using namespace slang;
 using namespace std::literals;
 
-static const char RelativeTestPath[] = "tests/file_tests/corpus/include";
+static const char RelativeTestPath[] = "/Volumes/hrt/surge/testing";
```

Comment:
Probably shouldn't commit this change, since other people don't necessarily check out to the same location.